### ISSUE TITLE
"Semi-Automatic” Weapon Attribute

### ISSUE THREAD

**Issue by Swampson** [queryluke/masseffect-5e#73]
This is not a comment on the actual rule (which I really like), but a piece of “gun nut” pedantry. The way this term is used in the rules differs drastically from the actual meaning of the term: https://en.wikipedia.org/wiki/Semi-automatic_firearm. I would suggest we change this attribute to “Rapid Fire”.

**Comment by queryluke** [comment]
Is the issue that this system uses thermal clips and a true "semi-automatic" weapon in this universe would reload the thermal clip for you?

I'm a complete gun novice, but in my mind I was thinking:

Single shot - guns that have some much power and kick that it takes a full action to ready/aim/fire.
Semi-automatic - guns that are light enough and have enough control that they can be fired twice in a single round, but each pull of the trigger only shoots a single "bullet"
3rb - Hold down trigger, shoots 3 "bullets"
automatic - hold down trigger, shoots multiple "bullets"

**Comment by Swampson** [comment]
IMO all of your firing modes are good and make sense. My thinking is that "semi-automatic" has a specific meaning for people who are knowledgable about firearms and/or may be bringing knowledge from other modern RPG settings.
If we use a term like "Rapid Fire" or "Double Tap" instead of Semi-Auto it should not affect the comprehension for gun novices and will prevent confusion for those that are already familiar with the "semi auto" term.
The more I think about it, the more I like "Double Tap" because it's more descriptive and specific - the weapon is fired twice in one action. "Rapid Fire" is more prone to be confused with with the "3RB" or "Automatic" terms.

**Comment by Swampson** [comment]
Sorry, I didn't answer your original question:
> Is the issue that this system uses thermal clips and a true "semi-automatic" weapon in this universe would reload the thermal clip for you?

So, in Mass Effect this analogy is a little tricky. In the real world, you typically load a magazine or "clip" into a firearm. The magazine contains a number of cartridges/bullets. With a semi-automatic weapon, some of the exhaust gas from the exploding gunpowder is used to operate a mechanism in the firearm that automatically ejects the spent bullet casing and loads a new round into the chamber, ready to fire. https://giphy.com/gifs/semi-E2jaorXMIQgdW

An example of a non semi-automatic weapon would be something like a bolt-action rifle or lever-action "cowboy" rifle, where the operator has to manually manipulate a mechanism to eject the spent casing and chamber a new round.

In Mass Effect, the actual projectile that's fired is one or more tiny pieces of metal shaved of a block of alloy housed within the weapon, so you're never actually reloading projectiles. By the strict definition, every small-arms weapon in the setting is either semi automatic or fully automatic. Re-loading thermal clips is pragmatically the same as re-loading a magazine of ammunition into a real world weapon, in that the gun can't fire until you've ejected the spent clip and loaded a fresh one.

**Short version: All weapons in ME are either semi-auto or full-auto, and the distinction between them has nothing to do with thermal clips but with how many "bullets" are fired each time the operator depresses the trigger:
- Semi-Auto: one "shot" per trigger squeeze
- 3RB: 3 "shots" per trigger squeeze
- Full-Auto: multiple (presumably > 3) "shots" per trigger squeeze**

**Comment by queryluke** [comment]
Got it. Yeah, let's switch it up to "Double Tap", I think I was worried that "Rapid Fire" would get conflated with "Automatic", but I like "Double Tap" as well.

**Comment by Swampson** [comment]
I've changed this to "Double Tap" in the Weapon Rules page, but don't see where to update it on the Google Sheets: I see the "Tags" column (K), but none of the weapons currently have "Semi-Automatic" listed as a tag. Should I be looking somewhere else to make this change?

**Comment by queryluke** [comment]
I think I've got the link to the 1.0 weapons page. Using a [different sheet now](https://docs.google.com/spreadsheets/d/132iB2ONH0BzlHUlGA9DRHMRP0aUu0I_dpQlJzSgoOXY/edit)

**Comment by Swampson** [comment]
Ah, cool. It looks like you've already done the Find & Replace for the Double Tap tag?

**Comment by queryluke** [comment]
yes, I've also pulled the changed into a separate branch I'm working on. But I didn't update anything on the Weapon Rules page.
